### DISCUSSION TITLE
Patrol Drive command stop working after patrol upgrade

### DISCUSSION THREAD

**Discussion by LolaYuz** [leancodepl/patrol#997] [Q&A] [answered]
I am writing all my tests in flutter, using patrol for native features automation.
Recently I ran the tests from the command line using "flutter drive --target blabla_test.dart" but after the upgrade, the command is not recognized. "flutter test" is also not running the tests.
 How to downgrade the version or what is the proper command to run the integration?
Thank you

**Comment by proximilian** [comment]
Hi, @LolaYuz 
Yes, after Patrol 1.0  `patrol drive` command is no longer available due to many reasons. You can use `patrol test --target <fileName_test>.dart`.  But `flutter test` should work ok. Please check your Flutter sdk path if is correct and check if the path is also correctly mentioned in your env. variables. Is there any error message thrown? Btw the command for upgrade `patrol` is `flutter pub upgrade patrol` and the command for upgrade `patrol_cli` is `patrol update`.

**Reply by bartekpacia** [reply]
`patrol drive` was deprecated since the beginning of this year. It was removed in mid-February in `patrol_cli` 1.0.

Please follow [the native automation setup guide](https://patrol.leancode.co/native/setup).

**Comment by bartekpacia** [comment, marked as answer]
`patrol drive` was deprecated since the beginning of this year. It was removed in mid-February in `patrol_cli` 1.0 in favor of `patrol test`.

Please follow [the native automation setup guide](https://patrol.leancode.co/native/setup) to make your project work with `patrol test`.
